azure/image/rhel/variables.pkr.hcl

```diff
@@ -118,3 +118,8 @@ variable "az_gallery_image_version" {
   type    = string
   default = ""
 }
+
+variable "config_script_src" {
+  type    = string
+  default = ""
+}
```

Comment:
Is this needed? can't we just drop this from variables file? same question for all  files

Reply:
The alternative is to have relative path (like https://github.com/confidential-containers/cloud-api-adaptor/blob/main/aws/image/ubuntu/aws-ubuntu.pkr.hcl#L43), which I wanted to avoid. Is there any other alternative ?

Reply:
I mean, if you already set config_script_src/others through the "-var" packer cmd option, is it needed also here?

Reply:
Ah I see. Afaik, the variables must be there in .pkr files for it to passed via command line